src/components/OptionPage.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import ImageItem from "./ImageItem";
```

Comment:
It just occurred to me that we might be reinventing the wheel, and sure enough, `react-bootstrap`(which is already included in the package) has an [`Image` component](https://react-bootstrap.github.io/components/images/). Take a look and see if this would be better. (If you do decide to use `react-bootstrap`, I'm so sorry for wasting your time. I kept googling without bootstrap and got nothing.)

Reply:
They even have `Card` and many other components.

Reply:
BTW I used `Button` and `ButtonGroup` in the popup (`index.html`).

Reply:
Okay, so that's why components were able to be nested without `{this.props.children}`

Reply:
I am going to leave `Card` as is, but replace image and maybe text with bootstrap components